proposals/acp/use-cases.md

```diff
@@ -48,17 +48,17 @@ Content-Type: text/turtle
 @prefix acp: <http://www.w3.org/ns/solid/acp#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
-<> a acp:AccessPolicyResource .
+<> a acp:AccessControlResource .
```

Comment:
Why do you type it as `acp:AccessControlResourc`? Does it mean that now one can control access to it by directly using `acp:access`? This also seems to go agains statement above:

> Access Policies and Access Rules can be defined in any resource.

Reply:
Looking at diff you had the same change in few more places. I only commented on the first one.

Reply:
Removed.